portal/client/src/pages/StudyList/index.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+import { Card, Level, Box, Button, ButtonGroup, Checkbox, TextBox, Icon, Columns, Table } from '../../components'
+
+import PageContainer from '../../components/composite/PageLayout'
+
+const StudyLists = () => {
+    const [isLoading, setLoading] = useState(false)
+    const [isSubmitting, setIsSubmitting] = useState(false)
+    const data = []
+    const columns = []
+
+    return (
+        <PageContainer
+            hasNoHeader
+            isLoading={isLoading}
+            title={'View study list'}
```

Comment:
You dont need the `{}` unless you have some variable or express in it. I see in many places you have used the `{}` just for the string, lets remove it, as this adds the overhead to JS runtime to evaulate string uncessarily